modular_citadel/code/modules/mob/living/silicon/robot/dogborg_equipment.dm

```diff
@@ -19,20 +20,23 @@ SLEEPER CODE IS IN game/objects/items/devices/dogborg_sleeper.dm !
 	desc = "The jaws of the law. Very sharp."
 	icon_state = "jaws"
 	force = 12
+	cellcost = 1000 // Secborg can only harm with the baton, why does the K9 get a free module to do 12 damage with?
 	attack_verb = list("chomped", "bit", "ripped", "mauled", "enforced")
 
+
 /obj/item/dogborg/jaws/small
 	name = "puppy jaws"
 	desc = "Rubberized teeth designed to protect accidental harm. Sharp enough for specialized tasks however."
 	icon_state = "smalljaws"
 	force = 6
+	cellcost = 500 // Less cell use for the smaller jaws.
 	attack_verb = list("nibbled", "bit", "gnawed", "chomped", "nommed")
 	var/status = 0
 
 /obj/item/dogborg/jaws/attack(atom/A, mob/living/silicon/robot/user)
 	..()
+	user.cell.use(cellcost)
 	user.do_attack_animation(A, ATTACK_EFFECT_BITE)
-	log_combat(user, A, "bit")

```

Comment:
This needs to be readded

Reply:
Ghommie told me it can be removed because all of attacks are automatically logged?

Reply:
Yea, log_combat() is already called on `/obj/item/attack()`